content/zh/docs/reference/glossary/aggregation-layer.md

```diff
@@ -42,5 +42,5 @@ tags:
 When you've configured the {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} to [support additional APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/), you can add `APIService` objects to "claim" a URL path in the Kubernetes API.
 -->
 
-当您配置了 {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} 来 [支持额外的 APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/)，您就可以在 Kubernetes API 中增 `APIService` 对象来  "声明" 一个 URL 路径。 
+当您配置了 {{< glossary_tooltip text="Kubernetes API Server" term_id="kube-apiserver" >}} 来 [支持额外的 APIs](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/)，您就可以在 Kubernetes API 中增加 `APIService` 对象来  "声明" 一个 URL 路径。 
```

Comment:
这里的 claim 翻译有问题。参见 claim 的对应翻译词条：

```
v. 要求（拥有）；索取；认领to demand or ask for sth because you believe it is your legal right to own or to have it
```

在这个上下文中，claim 不能翻译为“声明”。添加 `APIService` 对象的目的是 kubernetes API 的众多资源路径中占据一条路径下来，从所占据的这个路径 (URL Path) 访问 API 时，会有特殊的处理逻辑。

类似的，在 Persistent Volume Claim 中，Claim 也取同样的含义。都不能翻译为“声明”。

翻译为“认领“也不对，因为“认领”的意思是说某物本来就属于你，你只是要回了你的东西。

综合以上考虑，我建议翻译为“申领”。添加 APIService 是对一个 URL 路径的申领；PVC 是对一个 PV 的申领。“申领”表达是一种意愿和请求，不见得会导向最终的持有，因为各种策略的评估，最终很可能申领不成功，这是可以的。

供参考。

Reply:
FWIW，我对这个翻译有印象，看了一下 dashboard 的翻译，简体中文是「声明」繁体中文是「索取」，似乎是因为 https://github.com/kubernetes/dashboard/pull/2780 只改了简体。

我暂时不确定哪个词更好。

Reply:
那暂时就先用声明吧。

Reply:
肯定不能翻译成“声明”啊！

Reply:
有些专有的名词是不是不翻译更好？

Reply:
同意。
关键问题是判断哪些是专有名词。node 是不是？master node 是不是？deployment 是不是？server 是不是？

Reply:
这种有明确的中文对应的专有名词我建议翻译。
对应中文有多种含义的可以翻译加原文。
这里的claim大家意见不统一，那就用中文表达加原文。

Reply:
https://github.com/k8smeetup/k8s-official-translation/pull/7
I have put claim into the translation glossary sheet, please take a look!